### DISCUSSION TITLE
Theres so many complicated terms ...

### DISCUSSION THREAD

**Discussion by phasedanc3** [community/community#47396] [New to GitHub]
I just would like to know where do I click to my downloads on?.
a pull, a push, a repository, I thought a repository was sort of like a specialised treasure trove of downloads. I mean I fully expected I'd have to type in a hard command in a CLI to access the goods of the developers or whatever but yes.

dismayed, low self esteem, I've tried.

Cya later!

**Comment by melvincwng** [comment]
Hey @phasedanc3, learning Git or GitHub can be daunting for beginners at first. But don't worry, keep on learning and you will eventually master the usage of Git/GitHub.

The key is to stay persistent and keep on learning 👌 

For starters, you can visit this free and interactive course below to learn the basics of Git and GitHub: https://skills.github.com/

Going through this course would provide you the necessary knowledge and skills to start developing on this platform.

Take care and all the best in your learning journey 😃

**Comment by airtower-luna** [comment]
> I thought a repository was sort of like a specialized treasure trove of downloads.

A repository exists primarily to keep track of changes to the files inside, and share that history among developers. If all you want is to offer a few files for download a file sharing service might be better.

I highly recommend you take a look at the [Git Book](https://git-scm.com/book/en/v2) (it's a website, though you can also download an ebook). The first three chapters explain the idea of Git and basic usage. :slightly_smiling_face:

**Comment by queenofcorgis** [comment]
@phasedanc3 be gentle to yourself, you're learning a brand-new world, keep trying 👍🏼 ! Thanks @melvincwng and @airtower-luna for offering helpful resources. If either of them helped you - don't forget to mark as answer ✅ .
